src/styles/blocks/header.scss

```diff
@@ -0,0 +1,51 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding-right: 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  font-weight: 500;
+  font-size: 12px;
+
+  &__logo-moyo {
+    position: absolute;
+    margin: 10px 50px;
+    width: 40px;
+    height: 40px;
+  }
+}
+
+.nav__list {
+  display: flex;
+  list-style-type: none;
+  white-space: nowrap;
+}
+
+.nav__link {
+  display: flex;
+  color: #060b35;
+  cursor: pointer;
+  padding: 23px 0;
+  text-transform: uppercase;
+
+  &:hover {
+    color: $main-color;
+  }
+}
+
+.nav__li:not(:last-child) {
+  margin-right: 20px;
+}
+
+.is-active {
+  color: $main-color;
+
+  &::after {
+    content: "";
+    height: 4px;
+    background-color: $main-color;
+    border-radius: 8px;
+    width: 37px;
```

Comment:
it is a bad idea to hardcode the width. it should work properly if we set this class to another link